backend/src/index.js

```diff
@@ -302,9 +302,8 @@ async function main() {
         currentProposals = await extendWithTelemetryInfo(
           epochStats.currentProposals
         );
-        const onlineValidatingNodes = pickOnlineValidatingNode(
-          currentValidators
-        );
```

Comment:
It's done by precommit hook

Reply:
Run `npm install` from the root of the near-explorer project, so it installs the exact version of prettier on your side. Otherwise, it fails on CI. We will update the prettier and reformat the codebase in a separate PR.